### DISCUSSION TITLE
Trojita getting dropped from Gentoo repositories

### DISCUSSION THREAD

**Discussion by V10lator** [lxqt/lxqt#2058] [General]
See https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=a07069036c57202ffe99a2738b3a19e11c0a6823

So Gentoo (as well as Suse and maybe othe distros, too) users are left without an e-mail client on LXQT.

**Comment by wxl** [comment]
Trojita is not part of LXQt.

**Reply by tsujan** [reply]
@V10lator, please don't use the bug tracker for anything other than bug reports and feature requests!

As @wxl said, Trojita isn't LXQt's e-mail client. You can use any DE-agnostic email client with LXQt.

Personally, I can't use anything less than Thunderbird. I use it with KDocker (to put it inside the notification area -- there's also ~Firebird~ Birdtray for that but it has a huge memory leak) and an extension called "Minimize On Close".

EDIT: I wrote "Firebird" but meant "Birdtray"; corrected it.

**Reply by stefonarch** [reply]
@tsujan : time to merge the new template?

**Reply by tsujan** [reply]
@stefonarch 
Why not? Merge it and we'll see the effect.

**Reply by tsujan** [reply]
BTW, I think it may be better to clean up Discussions once in a while and remove topics that aren't relevant.

The problem is that there are different opinions about what's relevant and what's not. IMHO, topics that aren't related to LXQt at all can be removed after 6 months or so.

**Reply by stefonarch** [reply]
> Why not? Merge it and we'll see the effect.

Done, but maybe the bug report description should be more precise, we'll see if other stuff still slips trough.
I agree about keeping in order discussions, but it's not easy. No idea how good stuff there will be found by search engines.

**Reply by tsujan** [reply]
> No idea how good stuff there will be found by search engines.

You're right. I hadn't thought about that.

**Reply by stefonarch** [reply]
Btw, a good example is this discussion: it's about a qt mail client and _could_ involve LXQt, so it's not completely unrelated. Kind of "Qt related news"

**Reply by tsujan** [reply]
I agree.
